modules/vector-search/pages/run-vector-search-ui.adoc

```diff
@@ -102,6 +102,27 @@ Click btn:[View on GitHub] to view and copy the entire Vector Search query paylo
 Make sure you remove the lines for `// tag::partial[]` and `// end::partial[]`.
 ==== 
 
+[#base64]
+=== Example: Running a Semantic Search Query with a base64 Encoded String 
+
+If your database is running Couchbase Server version 7.6.2 or later, you can use vectors encoded as base64 strings with Vector Search. 
+For example, the following document describes the color `navy`, with base64 encoded strings in the `embedding_vector_dot` and `colorvect_l2` fields instead of arrays: 
+
+[source,json]
+----
+include::example$vector-search-base64-document-example.json[]
+----
+
+The following query uses a base64 encoded string for the same query as <<large,Running a Semantic Search Query with a Large Embedding Vector>> to return the document for `navy`: 
+
+[source,json]
+----
+include::example$run-vector-search-payload-base64-ui.jsonc[]
+----
+
+NOTE: You can only use base64 encoded strings in your Vector Search queries if your documents use base64 encoded strings, indexed with the *vector_base64* field data type.
```

Comment:
You can use base64 encoded strings in your Vector Search only if your documents use base64-encoded strings...